de-DE/docs/find.ndb

```diff
@@ -127,6 +127,7 @@ ellipticD(x,k)~Wert des elliptischen Integrals D(x,k)~elliptisches integral~elli
 ellipticE(x,k)~Wert des elliptischen Integrals E(x,k)~elliptisches integral~ellipticE(
 ellipticF(x,k)~ Wert des elliptischen Integrals F(x,k)~elliptisches integral~ellipticF(
 ellipticPi(x,n,k)~Wert des elliptischen Integrals Pi(x,n,k)~elliptisches integral~ellipticPi(
+endswith(STR,STR)~Gibt den Wert, True zur�ck, wenn der zweite Substring am Ende des ersten Strings gefunden wurde, sonst False

```

Comment:
Syntax error. It's best to edit this file using the NDB tools. This is missing the keywords and the terminal import string, e.g.:

`endswith(STR,STR)~Gibt den Wert, True zur�ck, wenn der zweite Substring am Ende des ersten Strings gefunden wurde, sonst False~endswith() string end termination~endswith(`